desktop-environment.el

```diff
@@ -182,6 +208,49 @@ portion of the screen."
   (message "New volume value: %s" (desktop-environment-volume-get)))
 
 
+;;; Helper functions - keyboard backlight
+(defun desktop-environment-keyboard-backlight-percent ()
+  "Return the new keyboard backlight value as a % instead
+of a single integer."
+  (if (eq (desktop-environment-keyboard-backlight-get) 0)
+      (progn
+        (let ((percent "0.0"))
+          (save-match-data percent)))
+  (progn (let ((percent (* (/ (* (desktop-environment-keyboard-backlight-get) 1.0)
+                            (* (desktop-environment-keyboard-backlight-get-max) 1.0)) 100)))
```

Comment:
would you mind changing the spacing of this expression so it becomes clearer what gets divided by what?

Reply:
Changed it for a clearer indentation.